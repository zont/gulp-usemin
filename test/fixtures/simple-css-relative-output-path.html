<!-- build:css /style.css -->
<link rel="stylesheet" href="css/clear.css"/>
<link rel="stylesheet" href="css/main.css"/>
<!-- endbuild -->