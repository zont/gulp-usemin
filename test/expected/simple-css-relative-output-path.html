<link rel="stylesheet" href="output-path/style.css"/>